src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -411,12 +410,6 @@ private async Task ProcessResourceChange<T>(WatchEventType watchEventType, T res
                         _logger.LogTrace("Updating application model resource {ResourceName} with {ResourceKind} resource {ResourceName}", appModelResource.Name, resourceKind, resource.Metadata.Name);
                     }
 
-                    if (_hiddenResources.TryAdd(appModelResource, true))
-                    {
-                        // Hide the application model resource because we have the DCP resource
-                        await notificationService.PublishUpdateAsync(appModelResource, s => s with { State = "Hidden" }).ConfigureAwait(false);
-                    }
```

Comment:
This isn't a small change. I like that we can remove it but I don't understand why.

Reply:
Yes, it's always right.